cycle-savings_vs_clusters.py

```diff
@@ -0,0 +1,18 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+water_size = [3,4,5,7,8,9,10]
+water = [3,2,2,2,1,1,1]
```

Comment:
i.e. `water_scf_savings = [...]`

Reply:
Was something like this file what you were asking for Ryan, or should the graph be delivered differently?

Reply:
Also, I presume we'll be needing similar graphs regarding density_times/supersystem_times vs cluster_sizes?

Reply:
This looks fine. You should set it up to make a PDF image of the graph. Yeah a plot of time savings vs cluster size would be good too.